README.md

```diff
@@ -93,7 +94,8 @@ export default () => (
   <tr><td>  response </td><td><code>any</code></td><td>Response from the resolved promise (`fn`)</td></tr>
   <tr><td>  error </td><td><code>any</code></td><td>Error from the rejected promise (`fn`)</td></tr>
   <tr><td>  load </td><td><code>(...args: any) => ?Promise&lt;any&gt;</code></td><td>Trigger to load `fn`</td></tr>
-  <tr><td>  hasResponseInCache </td><td><code>boolean</code></td><td>Returns `true` if data already exists in the context cache.</td></tr>
+  <tr><td>  hasResponseInCache </td><td><code>boolean</code></td><td>Returns `true` if data already exists in cache.</td></tr>
+  <tr><td>  cachedTimestamp </td><td><code>timestamp</code></td><td>Returns time since epoch of the cached response in localStorage.</td></tr>
```

Comment:
What's the use case of having a `cachedTimestamp`? I thought it was only really necessary to determine if a long-lived cache is invalid, however, our cache is short-lived as it loads on mount/on trigger.

Reply:
The user may want to clear the caches manually after a certain time. We haven't implemented any expiration stuff yet.